be/src/formats/parquet/column_chunk_reader.cpp

```diff
@@ -140,6 +140,9 @@ Status ColumnChunkReader::_read_and_decompress_page_data(uint32_t compressed_siz
     Slice read_data;
     auto ret = _page_reader->peek(read_size);
     if (ret.ok()) {
+        // peek dos not advance offset.
+        int64_t pos = _page_reader->get_offset();
+        _page_reader->seek_to_offset(pos + read_size);
```

Comment:
```suggestion
    RETURN_IF_ERROR(_page_reader->skip(ret->size());
```
The number of bytes actually returned by `peek()` may be less than the requested size, i.e, you should not assume `ret->size() == read_size` is always true.